extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js

```diff
@@ -1,13 +1,14 @@
 require([
 	'jquery',
+	'moment',
 	'wikia.tracker',
 	'wikia.ui.factory',
 	'wikia.mustache',
 	'wikia.window',
 	'wikia.throbber',
 	'embeddablediscussions.templates.mustache',
 	'EmbeddableDiscussionsSharing'
-], function ($, tracker, uiFactory, mustache, window, throbber, templates, sharing) {
+], function ($, moment, tracker, uiFactory, mustache, window, throbber, templates, sharing) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has 9 parameters which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)